src/libraries/Microsoft.Extensions.Configuration.Binder/src/ConfigurationBinder.cs

```diff
@@ -367,6 +298,37 @@ private static object BindToCollection(Type type, IConfiguration config, BinderO
                     return;
                 }
 
+                // for sets and read-only set interfaces, we concatenate on to what is already there
```

Comment:
This comment isn't true, right? `BindSet` creates a new instance.

Reply:
Well spotted - thanks!

Reply:
I also updated the comment re. dictionaries.